jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -2,12 +2,22 @@ lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
 
 pipeline {
     agent none
+    environment {
```

Comment:
Can you help add a timeout for this job? Ideally, it should have been added from when it was first created, but better late than never :)

Thank you!